p2p/discover/table.go

```diff
@@ -296,8 +297,9 @@ loop:
 			}
 		case <-revalidateDone:
 			revalidate.Reset(tab.revalidateInterval)
-			if tab.live() == 0 && len(waiting) == 0 && minRefreshTimer == nil {
+			if tab.live() == 0 && len(waiting) == 0 && minRefreshTimerActive.CompareAndSwap(false, true) {
```

Comment:
Does this check also work when `minRefreshTimerActive` is nil ?

Reply:
Should you also remove `minRefreshTimer` now , since it's not doing anything?

Reply:
it used for `defer timer.stop()`

Reply:
Right, I missed the code below .